index.js

```diff
@@ -1,7 +1,30 @@
-// Iteration 1: Names and Input
+const hacker1 = "Nishtha";
+console.log(`The driver's Name is ${hacker1}`);
+const hacker2 = "John";
+console.log(`The navigator's Name is ${hacker2}`);
+let i=hacker1.length;
+let j=hacker2.length;
+if (i>j)
+{
+  console.log(`The driver has the longest name, it has ${i} characters.`);
+}else if(j>i) {
+  console.log(`It seems that the navigator has the longest name, it has ${j} characters.`);
+}
+  else if(i=j){
+  console.log(`Wow, you both have equally long names,${j} characters!.`);
+  }
+  for( k = 0; k < hacker1.length ; k++ ) {
```

Comment:
Even inside the loops use the `let` keyword, so this variable has block scope instead of being a global one